lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to your CDK TypeScript project
+
+This is a blank project for CDK development with TypeScript.
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+## Useful commands
+
+* `npm run build`   compile typescript to js
+* `npm run watch`   watch for changes and compile
+* `npm run test`    perform the jest unit tests
+* `npx cdk deploy`  deploy this stack to your default AWS account/region
+* `npx cdk diff`    compare deployed stack with current state
+* `npx cdk synth`   emits the synthesized CloudFormation template
```

Comment:
If you could jolt down a quick REAMDE / TL;DR about the stack here would be good. Otherwise, advice to put `//TODO marker` to come back later, etc.

Reply:
AH, geez. You have already one at one level up. This can be removed.

Reply:
Resolved in d5ec8dbca4c8b9f443688c8ac38c1f373fac95be